pkg/querier/queryrange/queryrange.proto

```diff
@@ -169,3 +171,14 @@ message QueryResponse {
     QuantileSketchResponse quantileSketches = 8;
   }
 }
+
+message QueryRequest {
```

Comment:
am I missing something? we're not referencing/using this new message type anywhere afaict

Reply:
You are right! That should happen in a follow up. I think I originally thought I would use it. Will remove it.